src/languages/html.coffee

```diff
@@ -0,0 +1,681 @@
+helper = require '../helper.coffee'
+parser = require '../parser.coffee'
+
+parse5 = require '../../vendor/parse5/index.js'
```

Comment:
That sounds good to me.  While we wait for https://github.com/inikulin/parse5/pull/53, you can just reference your own branch in package.json and remove the files here.